AutoRest/AutoRest.Core.Tests/AutoRest.Core.Tests.csproj

```diff
@@ -80,6 +80,6 @@
     <PropertyGroup>
```

Comment:
if you like go extra mile of cleaning up, you can remove the "EnsureNuGetPackageBuildImports" according to "https://docs.nuget.org/consume/package-restore/migrating-to-automatic-package-restore". We have enabled the "nuget automatic restoration", this artifacts are useless